src/mpi/coll/bcast/bcast.c

```diff
@@ -12,6 +12,29 @@
 === BEGIN_MPI_T_CVAR_INFO_BLOCK ===
 
 cvars:
+    - name        : MPIR_CVAR_BCAST_ALGORITHM
+      category    : COLLECTIVE
+      type        : int
+      default     : 0
+      class       : device
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_ALL_EQ
+      description : >-
+        Variable to select bcast algorithm
+        0 - MPIR_Bcast_intra
+        1 - MPIR_GENERIC_TREE_Bcast with knomial trees
+        2 - MPIR_GENERIC_TREE_Bcast with kary trees
```

Comment:
This work is going to need to get integrated into the prereq patch that does manual algorithm selection (in progress).